README.md

```diff
@@ -80,15 +80,27 @@ cronjobs:
 With this in your current directory, you can now:
 
 ```sh
-# Bring up a local development environment using docker-compose
+# Bring up a local development environment using docker compose
 ktd up
+```
 
+```sh
 # Deploy the project to a Kubernetes namespace
 kubetools deploy my-namespace
 ```
 
 ## Installing
 
+**NOTE**: before upgrading to version 14.0 or above, you _must_ run `ktd destroy` for all existing 
+local Kubetools projects.
```

Comment:
I believe this note belongs in CHANGELOG.md because it's a mostly a matter for people upgrading from 13 to 14, and there shouldn't be anything to do special for any new project.

Reply:
Have moved this within this commit:
[7a8b920](https://github.com/EDITD/kubetools/pull/163/commits/7a8b920979214f3b3731fce5706bfd3f56f72628)